tsconfig.json

```diff
@@ -13,12 +13,13 @@
     "noImplicitAny": true,
     "outDir": "./lib",
     "paths": {
-      "*": ["node_modules/*"]
+      "*": ["node_modules/*", "src/*"]
```

Comment:
what does this do?

Reply:
wanted to make imports a la `NODE_PATH` work. `src` is included as path for module resolution. It worked but messed up jest badly, so I went back to relative imports. Good point, we can remove it again.

Reply:
removed